.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
```

Comment:
If you figure out how to test darwin on Travis, please send a PR to Closure Rules because we'd love to have that. Also I'm really glad to see the Travis config we built for CR is helping out other projects!